packages/components/src/components/flipcard/flipcard.stories.ts

```diff
@@ -57,16 +58,16 @@ export const Default = {
 };
 
 /**
- * Use the `variant` attribute to set the color variant for front or back.
+ * Use the `variant` attribute to set the color variant for front or back:
+ *
  * - empty (default)
```

Comment:
IMO all these options should be inside back ticks

Reply:
Updated